### DISCUSSION TITLE
prefix_and_default redirects default language page from "prefix" path to "no prefix" path

### DISCUSSION THREAD

**Discussion by darrenchiu** [nuxt-modules/i18n#1325] [Q&A]
I have a question on the prefix_and_default strategy. From the doc, it is saying that "you will get URLs with prefixes for every language, but URLs for the default language will also have a non-prefixed version."

But how come when I implemented this, the `/en` path always redirects to the no prefix page? (Given that `en` is the default language)

Is this intended behaviour?

**Comment by rchl** [comment]
Yes. The non-prefixed path is preferred.

I guess the advantage of that strategy is that visiting prefixed route still works while it probably wouldn't always work for the `prefix_except_default` strategy (although not sure now since redirecting code might have gotten smarter over time)
